dask/dataframe/tests/test_reshape.py

```diff
@@ -75,15 +71,11 @@ def test_get_dummies_kwargs():
     ds = dd.from_pandas(s, 2)
     res = dd.get_dummies(ds)
     assert_eq(res, exp)
-    tm.assert_index_equal(res.columns, exp.columns)
 
     # dummy_na
     exp = pd.get_dummies(s, dummy_na=True)
-
-    ds = dd.from_pandas(s, 2)
     res = dd.get_dummies(ds, dummy_na=True)
     assert_eq(res, exp)
-    tm.assert_index_equal(res.columns, pd.Index([1, 2, 3, 5, np.nan]))
```

Comment:
This `tm.assert_index_equal` call fails with 

```python
E       AssertionError: Index are different
E
E       Index classes are different
E       [left]:  CategoricalIndex([1, 2, 3, 5, nan], categories=[1, 2, 3, 5], ordered=False, dtype='category')
E       [right]: Float64Index([1.0, 2.0, 3.0, 5.0, nan], dtype='float64')
```

but all the other `tm.assert_index_equal` calls in this test pass on pandas 1.4.0. Why are we removing the other asserts? 

Alternatively, could we just update this line to use `pd.Index([1, 2, 3, 5, np.nan], dtype=res.columns.dtype)` instead?

Reply:
The index is already being checked in `assert_eq`. I imagine that this tests was just outdated.